roles/branch-validation/tasks/main.yml

```diff
@@ -68,17 +61,14 @@
     nics:
       - net-name: internal
     name: "validate-{{ item }}"
+    availability_zone: "nova:{{ item }}"
```

Comment:
What I meant was that the hint using `nova` may fail. `nova` might not be the availability zone. That hint is <az>:<host>, and it only works in clusters where `nova` is the AZ that every node exists in. That won't be the case for many 2.1.x stacks, which is why I suggested using variable data to determine the appropriate AZ hint to use.